src/main/charts/bamboo-agent/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: bamboo-agent
 description: A chart for installing Bamboo Data Center remote agents on Kubernetes
 type: application
 version: 1.7.1
-appVersion: 9.0.0
+appVersion: 9.0.1
```

Comment:
I think it's good to have the app version bot to update ChangeLog as part of the PR well to make the release process easier.

Reply:
@nghazali updating changelog is part of https://github.com/atlassian/data-center-helm-charts/pull/471 i.e. it'll be updated automatically as a new release is cut.